recipes/kennard-stone/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "kennard-stone" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/7a/ce/a4e3b09323ed9e7ae0054795927ba62d1d9bfc5d3e09f0d92b8706a50e06/kennard_stone-1.1.1.tar.gz"
+  sha256: 107b46fe5e654f482c3f4455fd5275e56491f47c1d02cb8ed8351fdfc4350338
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - numpy
+    - pandas
+    - pip
+    - python >=3.6
+    - scikit-learn
```

Nominally pure Python packages should only need `python` and `pip` in `host`:
```suggestion
    - pip
    - python >=3.6
```